include/deal.II/base/quadrature_lib.h

```diff
@@ -855,7 +865,8 @@ class QWitherdenVincentSimplex : public QSimplex<dim>
    * Constructor taking the number of quadrature points in 1D direction
    * @p n_points_1D.
```

Comment:
Would you mind removing this comment (as one needs the full information in the class definition in my opinion) and simply state
```suggestion
   * Constructor. See the general class description for details on the parameters.
```